functions/contact-us.js

```diff
@@ -0,0 +1,76 @@
+const admin = require('firebase-admin');
+const functions = require('firebase-functions');
+const uuidv1 = require('uuid/v1');
+const SENDGRID_KEY = functions.config().sendgrid.key;
+
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(
+	'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'
```

Comment:
We should keep this in a file ignored by github, paste the keys on slack for now until the repo's are private. IMO.